DOC-2181: fix documentation for TINY-10178 to the release notes. (#2966)
tinymce/tinymce-docs: modules/ROOT/pages/6.8-release-notes.adoc

=== Pasting underlined content from MS Word with PowerPaste results in empty rows being underlined
// TINY-10178
Previously, when running **Powerpaste 6.2.3** and copying content from Microsoft Word that contains text content such as the below:

[source, html]
----
<p><s>Sold out on tickets.</s></p>
<p>&nbsp;</p>
<p>Sorry!</p>
----

after pasting the content into the {productname} editor, the editor would **insert** a unexpected `<s>` tags around the `+&nbsp;+` such as `<p><s>&nbsp;</s></p>` which would render a visual `-` "strikethough" between the two paragraph's.

> 

Reviewer: I don't think this is accurate. A more accurate description would be:
```
When a user pastes content from Word that would be converted to:

[source, html]
 ----
 <p><s>Sold out on tickets.</s></p>
 <p><s>&nbsp;</s></p>
 <p>Sorry!</p>
 ----
 
The content in Word does not show an underline in the second line. It just shows an empty line.
In {productname} it shows an underline in the second line, which makes an inconsistency between what is rendered in Word and {productname}.
To fix this inconsistency when there is a paragraph with no text content except `+&nbsp;+` then PowerPaste would unwrap that paragraph so that it becomes:

[source, html]
 ----
 <p>&nbsp;</p>
 ----

Instead of:

[source, html]
 ----
 <p><s><u>&nbsp;</u></s></p>
 ----
 ```

Author: ```suggestion
Previously, when a user pasted content from Word, the pasted content would be converted to: 

[source, html]
----
<p><s>Sold out on tickets.</s></p>
<p><s>&nbsp;</s></p>
<p>Sorry!</p>
----

As  a consequence, two distinct behaviours were apparent.

. Firstly, the content displayed in Word did **not** render an underline for the second line, but only as an empty line.
. In {productname} the same content renders an underline in the second line.

To fix this , when a paragraph contains no content except for a `+&nbsp;+`, then PowerPaste would unwrap that specific paragraph so that it becomes: 

[source, html]
----
<p>&nbsp;</p>
----

Instead of: 

[source, html]
----
<p><s><u>&nbsp;</u></s></p>
----
```

=== Pasting underlined content from MS Word with PowerPaste results in empty rows being underlined
// TINY-10178
Previously, when a user pasted content from Word, the pasted content would be converted to: 

[source, html]
----
<p><s>Sold out on tickets.</s></p>

> <p><s>&nbsp;</s></p>

Reviewer: ```suggestion
<p><u>&nbsp;</u></p>
```

<p>Sorry!</p>
----

As  a consequence, two distinct behaviours were apparent.

. Firstly, the content displayed in Word did **not** render an underline for the second line, but only as an empty line.
. In {productname} the same content renders an underline in the second line.

To fix this , when a paragraph contains no content except for a `+&nbsp;+`, then PowerPaste would unwrap that specific paragraph so that it becomes: 

[source, html]
----
<p>&nbsp;</p>
----

Instead of: 

[source, html]
----

> <p><s><u>&nbsp;</u></s></p>

Reviewer: ```suggestion
<p><u>&nbsp;</u></p>
```
